components/prism-io.js

```diff
@@ -0,0 +1,31 @@
+Prism.languages.io = {
+    'comment': [
```

Comment:
I'm reviewing from my phone so I can't actually check, but can you make sure you're using tabs for indentation in this file please?